src/xiana/swagger.clj

```diff
@@ -0,0 +1,221 @@
+(ns xiana.swagger
+  (:require
+    [clojure.string :as str]
+    [hiccup.core :as h]
+    [jsonista.core :as json]
+    [malli.util]
+    [meta-merge.core :refer [meta-merge]]
+    [reitit.coercion :as rcoercion]
+    [reitit.coercion.malli]
+    [reitit.core :as r]
+    [reitit.ring :as ring]
+    [reitit.swagger]
+    [reitit.trie :as trie]
+    [ring.util.response]))
+
+(defonce all-methods
+  [:get :patch :trace :connect :delete :head :post :options :put])
+
+(def routes->routes'
```

Comment:
Huge cognitive load trying to work out what this does. Would be easier, I think, if parts of these lets were extracted into carefully named fns.

Reply:
I believe, it's solved with the last changes. Thank you!